janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server-berkeleyje-es.yaml

```diff
@@ -0,0 +1,43 @@
+host: 0.0.0.0
+port: 8182
+scriptEvaluationTimeout: 30000
+channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
+graphs: {
+    graph: conf/gremlin-server/janusgraph-berkeleyje-es-server.properties}
+scriptEngines: {
+    gremlin-groovy: {
```

Comment:
This is unfortunately a YAML file for TinkerPop >= 3.3.0, but the 0.2 branch is compatible with TinkerPop 3.2.z. So, this YAML file won't work there.
You either have to create different PRs for the different branches (0.2 and 0.3 from where we can merge to master) or you just change the base branch to 0.3. Since we decided that we don't want to target 0.2 with any new pull requests and this is technically a new PR, I think it's ok to just target 0.3 here.

Reply:
@FlorianHockmann I've made a re targeted the PR to 0.3. 
Also, I've added a header into `gremlin-server-berkeleyje-es.yaml` because all `yaml` files contain it in 0.3..

Reply:
Oh yes, didn't even see the missing license header. I probably added that in the other files when I enabled the license header enforcement.